opat/drupal-9.yaml

```diff
@@ -746,5 +746,492 @@ chapters:
           - name: "web"
             adherence:
               level: "not-applicable"
+  # Note that this will likely be replaced with an automated process.
+  functional_performance_criteria:
+    notes: Not applicable.
+    criteria:
+      - num: "302.1"
+        components:
+          - name: "none"
+            adherence:
+              level: "supports"
+              notes: Testing has been done with JAWS, NVDA and VoiceOver.
+      - num: "302.2"
+        components:
+          - name: "none"
+            adherence:
+              level: "supports"
+              notes: Testing has been done with browser zoom and ZoomText.
+      - num: "302.3"
+        components:
+          - name: "none"
+            adherence:
+              level: "supports"
+              notes: The interface has been reviewed for use of color.
+      - num: "302.4"
+        components:
+          - name: "none"
+            adherence:
+              level: "not-applicable"
+      - num: "302.5"
+        components:
+          - name: "none"
+            adherence:
+              level: "not-applicable"
+      - num: "302.6"
+        components:
+          - name: "none"
+            adherence:
+              level: "not-applicable"
+      - num: "302.7"
+        components:
+          - name: "none"
+            adherence:
+              level: "supports"
+              notes: Drupal's interface does not restrict users with limited manipulation.
+      - num: "302.8"
+        components:
+          - name: "none"
+            adherence:
+              level: "supports"
+              notes: Drupal's interface does not restrict users with limited reach or strength.
+      - num: "302.9"
+        components:
+          - name: "none"
+            adherence:
+              level: "not-applicable"
+  # Chapter 4
+  # This is included as an exmple. It should not need to be included for exclusively web based ACRs.
```

Comment:
@mgifford since [Drupal VPAT](https://github.com/CivicActions/accessibility/blob/main/VPAT/Drupal9AccessibilityConformanceReport.md#chapter-4-hardware) only lists a note for hardware and software we shouldn't add all the criteria here and just add it to the catalog since you already collected it. That is my thought. What do you think?